plugins/services/src/js/containers/service-connection/ServiceConnectionEndpointList.tsx

```diff
@@ -44,8 +43,8 @@ class ServiceConnectionEndpointList extends React.Component {
     return getDisplayValue(protocol);
   }
 
-  getHostPortValue(portDefinition, service) {
-    if (!service.requirePorts) {
+  getHostPortValue(portDefinition) {
+    if (!portDefinition.hostPort) {
```

Comment:
can we follow up on this change down below?

it looks like we now can condense that to the following, right?

```typescript
  function getHostPortValue({ hostPort }) {
    return hostPort ? (
      <EndpointClipboardTrigger command={getDisplayValue(hostPort.toString())} />
    ) : (
      <Trans render="span">Auto Assigned</Trans>
    );
  }
```

Reply:
@janisz any thoughts on this?